testCourse/elements/pl-truth-table/pl-truth-table.css

```diff
@@ -0,0 +1,23 @@
+.pl-truth-table-container {
+    margin: 1em;
+    padding: 1ex;
+}
+
+.pl-truth-table-container table {
+	border: solid 1px gray;
+}
+
+.pl-truth-table-container tr:first-child {
+	background-color: #ffc107;
+	font-weight: bold;
+	text-align: center;
+}
+
+.pl-truth-table-container tr:nth-child(even) {
+	background-color: #f2f2f2;
+}
+
+.pl-truth-table-container td {
+    padding: 1ex;
+    margin: 1em;
+}
```

Comment:
The element files shouldn't be duplicated in `testCourse`, they should just be in the `elements` folder in the root directory.